gnosis/eth/ethereum_client.py

```diff
@@ -1447,8 +1448,10 @@ def deploy_and_initialize_contract(
 
                 if not contract_address:
                     contract_address = ChecksumAddress(
-                        Web3.toChecksumAddress(
-                            mk_contract_address(tx["from"], tx["nonce"])
+                        to_checksum_address(
```

Comment:
Can you create a `mk_contract_address` on `gnosis.eth.utils`? This is a little difficult to read